exporter/splunkhecexporter/logdata_to_splunk.go

```diff
@@ -24,6 +24,13 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/splunk"
 )
 
+const (
+	// spanIDFieldKey is the key used in log event for the span id (if any).
```

Comment:
Link to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/overview.md#trace-context-in-legacy-formats can be useful to explain the choice of keys.